PURL-TYPES.rst

```diff
@@ -255,6 +255,37 @@ rpm
       pkg:rpm/fedora/curl@7.50.3-1.fc25?arch=i386&distro=fedora-25
       pkg:rpm/centerim@4.22.10-1.el6?arch=i686&epoch=1&distro=fedora-25
 
+swift
+----
+``swift`` for Swift packages:
+
+- There is currently no default repository for Swift packages.
+- The ``namespace`` is used for the scope of a Swift package.
+  It is not case sensitive and must be lowercased.
+
+  - A package scope consists of alphanumeric characters and hyphens.
+    Hyphens may not occur at the beginning or end,
+    nor consecutively within a scope.
+  - The maximum length of a package scope is 39 characters.
+  - A valid package scope matches the following regular expression pattern
+    ``\A[a-zA-Z0-9](?:[a-zA-Z0-9]|-(?=[a-zA-Z0-9])){0,38}\z``
+
+- The ``name`` is the name declared in the Swift package manifest.
+  It is case and normalization insensitive,
```

Comment:
Same question with case sensitivity as with namespace.

Reply:
A package name, like a namespace, is case-insensitive. It's also normalization-insensitive to accommodate the greater range of characters allowed for names, relative to namespaces.